crates/e2e/macro/src/config.rs

```diff
@@ -64,22 +63,21 @@ impl Node {
 }
 
 /// The runtime emulator that should be used within `TestExternalities`
-#[cfg(any(test, feature = "sandbox"))]
 #[derive(Clone, Eq, PartialEq, Debug, darling::FromMeta)]
-pub enum RuntimeOnly {
-    #[darling(word)]
-    #[darling(skip)]
-    Default,
-    Sandbox(syn::Path),
+pub struct RuntimeOnly {
```

Comment:
This was a bit tricky: since we don’t want to import `ink_sandbox` in this crate, we can’t provide it as a default. It always has to be specified explicitly by the user.

Reply:
The `ink_e2e` machinery already has a good look at the manifest (or more specifically cargo metadata) to do stuff like compiling the root contract and it's contract dependencies (if any) (see [here](https://github.com/use-ink/ink/blob/96434d9ff94f9bc13372c238a76905a866d9e2f7/crates/e2e/src/contract_build.rs#L46-L58)).  
So we can do something there to bring back defaults (after this PR), while also providing user friendly errors when `ink_sandbox` is missing. 
I'm already planning to do something similar for when the user runs `cargo test` (not ABI aware) instead of `cargo contract test` (ABI aware), so I can look at this as well while doing that.